target/linux/ath79/dts/qca9531_tplink_tl-wr902ac-v1.dts

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "tplink,tl-wr902ac-v1", "qca,qca9531";
+	model = "TP-Link TL-WR902AC v1";
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+		label-mac-device = &wmac;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan2g {
+			label = "tp-link:green:wlan2g";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		qss {
+			label = "tp-link:green:qss";
+			gpios = <&gpio 10 GPIO_ACTIVE_LOW>;
```

Comment:
ar71xx has LED called WPS with gpio 0

Reply:
Renamed. But a migration script will still be needed, as board name is no longer used as LED name prefix.

Reply:
Tested? Which one matches the device, wps or qss?

Reply:
It doesn't have label on itself, only symbol, so let's keep what was set in ar71xx.

Reply:
Also, tested on HW as noted below.